contrib/etc/my.cnf

```diff
@@ -0,0 +1,12 @@
+[mysqld]
+
+# Disabling symbolic-links is recommended to prevent assorted security risks
+symbolic-links = 0
+
+# http://www.percona.com/blog/2008/05/31/dns-achilles-heel-mysql-installation/
+skip_name_resolve
+
+# http://www.chriscalender.com/ignoring-the-lostfound-directory-in-your-datadir/
+ignore-db-dirs=lost+found
```

Comment:
I believe this is deprecated, it should be ignore_db_dirs (?). There's a startup warning